sprocs/index.js

```diff
@@ -103,21 +104,25 @@ module.exports = {
             'instance_questions_lock.sql',
             'instance_questions_ensure_open.sql',
             'instance_questions_select_variant.sql',
+            'instance_questions_update_in_grading.sql',
+            'instance_questions_update_for_manual_grading.sql',
             'instance_questions_update_score.sql',
+            'instance_questions_manually_grade.sql',
+            'instance_questions_manually_grade_submission.sql',
             'instance_questions_next_allowed_grade.sql',
+            'instance_questions_select_manual_grading_objects.sql',
             'submissions_lock.sql',
             'submissions_select.sql',
             'submissions_insert.sql',
             'submissions_update_parsing.sql',
             'assessment_instances_update.sql',
-            'instance_questions_update_in_grading.sql',
-            'instance_question_select_manual_grading_objects.sql',
             'assessment_instances_close.sql',
             'grading_job_status.sql',
             'grading_jobs_lock.sql',
             'grading_jobs_insert.sql',
             'grading_jobs_insert_external_manual.sql',
             'grading_jobs_insert_internal.sql',
+            'grading_jobs_insert_manual.sql',
```

Comment:
I think it would be a good idea in terms of naming. If the spreadsheet manual grading upload is not using that SPROC, which I do not believe it is, I think it would make a lot of sense to rename the SPROC.

Reply:
Yeah, looks good. The instructor assessment question score upload calls `instance_questions_update_score.sql` directly and bypasses `grading_jobs_insert_external_manual.sql`. I think this will be okay.

Reply:
This has been done, but I also did not put in a new delegate call to the `grading_jobs_insert_manual.sql` SPROC, as it is called using different arguments. I realize that `internal` and `external` also use different arguments, so we can move the `manual` delegate call to the main `grading_jobs_insert` file, but it is unclear if `Manual` will remain a grading_method enum in the next project. It may turn into an action type that can be performed in `internal` and `external` grading types.

Just let me know if you want the delegate call moved into the `grading_jobs_insert` file. That can still be done.